### DISCUSSION TITLE
Retract a section when I open another section

### DISCUSSION THREAD

**Discussion by Marlony-Gabriel** [squidfunk/mkdocs-material#5335] [Q&A] [answered]
Hello people.
Is there any configuration that I can do to retract an opened section when I open another section on the navigation menu ?
In the image bellow I have three sections open: 1, 2 and 3
What I want is when I open section 2 for example, sections 1 and 3 are automatically retracted. In another words, I want only one section open at a time.

<img width="170" alt="cah" src="https://user-images.githubusercontent.com/129212251/230422604-8d4a02da-66da-4ca5-bbcd-6fddfbaf917a.png">

**Comment by squidfunk** [comment, marked as answer]
Navigation sections are closed when navigating, i.e., only the active path is opened, unless [`navigation.expand`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-expansion) is set. If you want to close the section on expansion/collapsing without navigation, you could implement this yourself with [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript).

**Reply by Marlony-Gabriel** [reply]
Thank you @squidfunk.

**Reply by pedro-psb** [reply]
Hey @Marlony-Gabriel , did you come up with a good JS snippet for this? :sweat_smile:
